### ISSUE TITLE
Problem about training on the GPU

### ISSUE THREAD

**Issue by LoveYourGang** [UrbComp/DeepTTE#13]
I ran the code on the pytorch-cpu-windows-py37 platform successfully, but when tring to use GPU, I have some problem. While training, the first epoch was normal, but an error occured when the second epoch started.
![250](https://user-images.githubusercontent.com/34276640/53718331-d2d90d00-3e95-11e9-9deb-03e0bfb989c1.jpg)
I don`t understand why "cudnn RNN backward can only be called in training mode", I`m sure I was training the model, because the "task" argument was "train". Error occurs only on GPU with the same code.

**Comment by LoveYourGang** [comment]
I have removed the error.

**Comment by LoveYourGang** [comment]
The code "model.train()" should be in the loop of the train. After each epoch, the "model.eval()" in the "evaluate()" function changes the mode.
